src/ServicePulse.Host/vue/src/assets/footer.css

```diff
@@ -1,87 +1,102 @@
 footer {
-    color: #bcc6c2;
-    font-size: 14px;
-    font-weight: normal;
-    height: 35px;
-    margin-top: 100px;
-    padding: 5px 0;
-    text-align: left;
-    text-align: center;
-    width: 100%;
-    background-color: #fff;
-    position: fixed;
-    bottom: 0;
-    border: 1px solid #ddd;
-    box-shadow: 0px -10px 16px #f2f6f7;
-    padding-top: 7px;
+  color: #bcc6c2;
+  font-size: 14px;
+  font-weight: normal;
+  height: 35px;
+  margin-top: 100px;
+  padding: 5px 0;
+  text-align: left;
+  text-align: center;
+  width: 100%;
+  background-color: #fff;
+  position: fixed;
+  bottom: 0;
+  border: 1px solid #ddd;
+  box-shadow: 0px -10px 16px #f2f6f7;
+  padding-top: 7px;
+  z-index: 999; /* this needs to fight with vue-highlight-code which forces z-index to 700 */
```

Comment:
Where is this 700?
I looked for it in https://github.com/search?q=repo%3Aelisiondesign%2Fvue-code-highlight+z-index&type=code

Reply:
I think we're using https://github.com/F-one-1/vue-highlight-code
It [shows a z-index of 1000](https://github.com/F-one-1/vue-highlight-code/blob/c5b4e848292453ee29cb9f0d8ab07bfeb51cc3c2/src/components/HighCode.vue#L281) but it also does not have a tag for the current version so I am not sure where npm is getting it from

Reply:
the z-index is being added into the inline style of the DOM element, not as part of imported CSS

Reply:
line 178 https://github.com/F-one-1/vue-highlight-code/blob/c5b4e848292453ee29cb9f0d8ab07bfeb51cc3c2/src/components/HighCode.vue#L178C21-L178C21